### ISSUE TITLE
/v1/producer/create_snapshot to return information in more usable way

### ISSUE THREAD

**Issue by matthewdarwin** [EOSIO/eos#9242]
/v1/producer/create_snapshot returns a JSON document that looks like this:

```
{"head_block_id":"00f1096aa8794edf06783c74cb5be2d4f5062da81cad00a96f2747a2a43b237d","snapshot_name":"...../snapshot-00f1096aa8794edf06783c74cb5be2d4f5062da81cad00a96f2747a2a43b237d.bin"}
```

This information does contain the block number, but you need to decode it from the head_block_id.  This is not intuitive, and possible not future safe.

So ideally there should be 2 new items added to this list:

- block_number
- chain_time

/v1/chain/get_info contains this info, so copy that idea for implementing this. Makes it easier to know when the details of the snapshot.

**Comment by matthewdarwin** [comment]
Also should return the snapshot version number, seeing as we're now getting to v4 snapshots.

https://github.com/EOSIO/eos/pull/9371/files/a5c8fa0b6c429c0ce0dc4b844b8a0205f647406f..ce6576b215f5d6ae2f106a27ef4416936913828b

**Comment by sanaraufx** [comment]
Fixed in 2.1. Let us know if you have any other questions.
